tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/compute/ComputeTests.java

```diff
@@ -19,7 +19,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-import java.awt.Color;
+import java.awt.*;
```

Comment:
unfold imported classes

Reply:
Yes, sorry the code formatter was filtering for awt classes.